### DISCUSSION TITLE
Problem with my compilation on FlutterFlow with Apple

### DISCUSSION THREAD

**Discussion by alexgranadosl** [codemagic-ci-cd/codemagic-docs#2088] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hello community! I am having a problem uploading my build from FlutterFlow with codemagic to Apple, the following message highlights the error:

Error (Xcode): Provisioning profile "Grupo Alarmas App OFICIAL ios_app_store 1696379342" doesn't match the entitlements file's value for the com.apple.developer.in-app-payments entitlement. Profile qualification is using entitlement definitions that may be out of date. Connect to network to update.
/Users/builder/clone/ios/Runner.xcodeproj

This error I have not been able to fix, my understanding is that it refers to an error in the processing of my provisioning profile, identifiers or certificates in question to payment processing, but in looking for a solution, I have not been able to find one.

I hope someone can help me, thank you very much!

### Expected results

I hope someone can help me to solve the problem as soon as possible.

### Actual results

This problem to be present and I want to solve today

**Comment by himesh-cm** [comment]
Hi @alexgranadosl, Please check if you have enabled Apple in-app payments for your `identifier` on your Apple developer account.

**Comment by himesh-cm** [comment]
Hi @alexgranadosl,
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
